### ISSUE TITLE
How to download

### ISSUE THREAD

**Issue by user18081972** [gyrovorbis/libevmu#2]
I want to run some VMU games on Windows. But this github page doesnt have a Windows x86_64 binary to download and run.
Is this a libretro core, or do I have to build it myself, or am i missing the point?

**Comment by gyrovorbis** [comment]
So sorry for the late response!!!

The truth is you basically just caught me with my pants down. I'm so so close to finishing up a new release for the emulator, but I also really wanted to go ahead and get the core open-sourced so that people can begin working on porting it to other platforms in parallel with me finishing up the release.

So basically, yeah, there's no link or official release to the download yet, as this is just the core library, not the whole application...

But it's cool, I just made a prerelease build that has been tested quite a bit and is pretty stable and has a whole bunch of features already implemented. Here's a link to it: https://www.dropbox.com/scl/fi/jhx5nbgth21v14cdsp25k/ElysianVMU-v1.5.0-prerelease.zip?rlkey=d9zajqhm0ltde5n26gvlme1qy&dl=0

Let me know if you have any problems or questions. If something screws up for you, please feel free to post another issue or feature request.

**Comment by user18081972** [comment]
Awesome! Thanks, ive been having fun playing Chao Adventure, now I don't have to keep replacing my VMU battery so much!
I can't wait for TCP/IP VMU-to-VMU to be implemented!
